com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoopDef.hlsl

```diff
@@ -152,15 +152,18 @@ float4 SampleEnv(LightLoopContext lightLoopContext, int index, float3 texCoord,
             color.rgb = SAMPLE_TEXTURECUBE_ARRAY_LOD_ABSTRACT(_EnvCubemapTextures, s_trilinear_clamp_sampler, texCoord, _EnvSliceSize * index + sliceIdx, lod).rgb;
         }
 
+        // Planar and Reflection Probes aren't pre-expose, so best to clamp to max16 here in case of inf
+        color.rgb = ClampToFloat16Max(color.rgb);
```

Comment:
That's wrong, you must clamp after 

color.rgb *= rangeCompressionFactorCompensation;

 // Planar and Reflection Probes aren't pre-expose, so best to clamp to max16 here in case of inf
        color.rgb = ClampToFloat16Max(color.rgb);
        
        otherwise rangeCompressionFactorCompensation become fully useless... rangeCompressionFactorCompensation is to currentlty allow to fit large range.